tests/test_database.py

```diff
@@ -28,11 +38,12 @@ def test_nn_to_linked_nn():
     nodes = database_client.nn_to_linked_nn(2699)
     print(nodes)
     assert nodes == [844, 2090, 2645]
+    
 
 def test_nn_to_linked_nn_too_large():
     nodes = database_client.nn_to_linked_nn(100000)
     print(nodes)
     assert nodes == []
 
 if __name__ == '__main__':
-    test_address_to_nn()
\ No newline at end of file
+    test_email_to_nn_no_node()
```

Comment:
Why not have this file always run every test function? Better yet, we could wrap this all up in PyTest for nice test reports

Reply:
@Andrew-Dickinson It does use pytest, that's what runs in the test phase of the CI.  I just run them manually sometimes since the console output is cleaner.

Reply:
To get reports we'd have to copy the results out of the container with another volume.  At work I considered doing this with GitLab, but it didn't look like the formatting was much better than just looking at the terminal.